### ISSUE TITLE
Ensures links have discernible text (.dropzone-showbox).

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#145]
User Experience:
If, the link does not have discernible text defined then the inaccessible link elements pose barriers to accessibility, as they are a fundamental component of a website. Users who rely exclusively on a keyboard (and no mouse) to navigate a webpage can only click on links that can receive programmatic focus. 

Note: User credentials should NOT be included in the bug.

Prerequisites:

1. Open Narrator via pressing ctrl+window+enter key.

2. Open Accessibility insights for web tool and select fast pass option.

 

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the Video player and select it.
8. Navigate to the various controls of the "Video player" dialog box.
9. Navigate to link control available below the Source button.
10. Open Accessibility insights for web tool and select fast pass option.
11. Verify, whether the  link does have discernible text or not.
 

Actual Result:

On the video player dialog box, link does not have discernible text defined. Instead Screen Reader announce link only.
 

Expected Result:

On the video player dialog box, link should have discernible text defined. So, that the Screen Reader user does able to get the proper information of the link control.

Note:

1. Similar issue exist with Chrome+JAWS+NVDA and edge+Narrator.
 

MAS Reference:

MAS 4.1.2 - Name, Role, Value (40B)

Issue: Ensures links have discernible text (link-name - https://dequeuniversity.com/rules/axe/3.3/link-name?application=msftAI)

Target application: API Management - https://accessibility.developer.azure-api.net/apis

Element path: .dropzone-showbox

Snippet: <a href="#" class="dropzone-showbox" alignment="center center"> <div class="fit background-preview"></div> </a>

How to fix: 
Fix all of the following:
Element is in tab order and does not have accessible text

Fix any of the following:
 Element does not have text that is visible to screen readers
 aria-label attribute does not exist or is empty
 aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
 Element's default semantics were not overridden with role="presentation"
 Element's default semantics were not overridden with role="none"

This accessibility issue was found using Accessibility Insights for Web 2.14.1 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

![image](https://user-images.githubusercontent.com/2320302/79780376-af1e9380-82f0-11ea-920f-b7e9d4201f36.png)

**Comment by azaslonov** [comment]
Fixed.
